.github/workflows/autofix.yml

```
@@ -44,19 +41,14 @@ jobs:
+            git fetch origin main
           changed_files=$(git diff --name-only --diff-filter=ACMRUXB origin/main)
-          
-          if [ -z "$changed_files" ]; then
-            echo "No changed files to lint."
-          elif echo "$changed_files" | grep -q "^\.eslintrc\.json$"; then
-            yarn lint:fix
-          else
-            echo "$changed_files" | grep -E "(.js$|.jsx$|.ts$|.tsx$)" | xargs -r yarn lint:fix
-          fi
-
+          echo "Changed files: $changed_files"
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 49: mapping values are not allowed in this context

(syntax-check)